protected/humhub/modules/user/models/Invite.php

```diff
@@ -21,16 +20,13 @@
  * @property string $email
  * @property string $source
  * @property string $token
- * @property string $created_at
- * @property integer $created_by
- * @property string $updated_at
- * @property integer $updated_by
  * @property string $language
  * @property string $firstname
  * @property string $lastname
  */
-class Invite extends ActiveRecord
+class Invite extends \humhub\db\HActiveRecord
```

Comment:
Removed `FileManager` functional for Invite model.
Meta information about created_..., updated_... available by `\humhub\db\traits\MetaActiveRecordTrait`